### DISCUSSION TITLE
Alternative channel for email?

### DISCUSSION THREAD

**Discussion by yellow1912** [chatwoot/chatwoot#1188] [Q&A]
**Is your feature request related to a problem? Please describe.**
When we integrate chatwoot with a platform that already has built in CRM of some sort, the email sending and syncing etc may already happen on that specific CRM. 

The system I'm working on for example has a nice integration with Google's gmail api that does allow sending and pulling emails nicely. All the information is basically stored already on the locally available CRM platform.

**Describe the solution you'd like**
It would be better to have an option to perhaps use an API channel to do what the current email channel does. It allows adding support for many local, specialized internal, external communication channels much easier?

**Comment by pranavrajs** [comment]
@yellow1912 I don't understand the issue. Let me tell you what I have understood so far.

1. You have a CRM with Gmail integration.
2. You would see emails coming in CRM inside Chatwoot.

Are you looking for an API channel? If so, can you please take a look at https://www.chatwoot.com/docs/channels/api/configure
